internal/app/plan.go

```diff
@@ -133,6 +133,12 @@ func releaseWithHooks(cmd orderedCommand, wg *sync.WaitGroup, sem chan struct{},
 		wg.Done()
 		<-sem
 	}()
+	if cmd.targetRelease != nil && !flags.destroy {
```

Comment:
@luisdavim I mean this specific part. Don't you want it to be `if cmd.targetRelease == nil && !flags.destroy` if the error then says `fmt.Errorf("nil target release")`?

Reply:
yes, this is checking if the target release is `nil` but only returns the error if the destroy flag is not set so we still try to run the uninstall command, keeping the previous behaviour for deletions.
I can remove the `&& !flags.destroy` part but the idea is to try to take the safest path...

Reply:
sorry, I was blind and couldn't see the `!=` vs `==` :(